python/paddle/fluid/tests/unittests/white_list/check_shape_white_list.py

```diff
@@ -31,4 +31,6 @@
     'multi_dot',
     'index_add',
     'reshape2',
+    'squeeze',
```

Comment:
why add to this?

Reply:
without this change, squeeze needs the input tensor to have least 100 elements. otherwise, the ut will fail before running into kernel.